docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2807,6 +2807,284 @@ have not been included. These are:
 #. We sample 100% of events. This is likely unrealistic, but the percent is probably very high still. 
 #. There are errors in SSN data, which are not modeled here. 
 
+Noise Functions
+^^^^^^^^^^^^^^^
+
+In order to have a realistic challenge with PRL, it is essential to have 
+noise added to the data. Noise is generally divided into two types: 
+
+#. **Column based noise:** errors in individual data entry, such as miswriting or incorrectly selecting responses 
+#. **Row based noise:** errors in the inclusion or exclusion of entire rows of data, such as duplication or omission 
+
+To begin, we will start with defining column based noise. Some general rules 
+for all column based noise include: 
+
+- The amount of each type of noise is individually configurable for each observer up to the max listed in the table below. This means that the end user can select that: census first names have 5% typographic errors. The minimum noise is 0 for all column noise. 
+- Simulants are selected for noise at random. Selection is not based on any attributes and simulants do not have a higher or lower propensity for noise that would carry across observers (e.g., there are not "messy" simulants who make errors on all forms). 
```

Comment:
Can you explain a bit more explicitly that a "noise level" is the probability of selecting each row to have that form of noise and that the selection for each form of noise is independent?

Also, we will need to define the order in which these noise types are applied. I don't think it matters much, but it will need to be chosen at some point so we might as well specify it now.

Reply:
Okay, I expanded and added order to this